src/coreclr/src/nativeaot/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -0,0 +1,559 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- Note that various places in SPCL depend on this resource name i.e. TplEventSource -->
+  <ItemGroup Label="Embedded Resources">
+    <EmbeddedResource Include="src\Resources\Strings.resx">
+      <GenerateSource>true</GenerateSource>
+      <ClassName>System.SR</ClassName>
+    </EmbeddedResource>
+  </ItemGroup>
+
+  <PropertyGroup>
+    <DefineConstants>SYSTEM_PRIVATE_CORELIB;ES_BUILD_PN;ES_SESSION_INFO;$(DefineConstants)</DefineConstants>
```

Comment:
```suggestion
    <DefineConstants>SYSTEM_PRIVATE_CORELIB;$(DefineConstants)</DefineConstants>
```

Reply:
`ES_SESSION_INFO` can be deleted from the sources too. The ifdef can be just `#if !ES_BUILD_STANDALONE && TARGET_WINDOWS`